src/Symfony/Component/Form/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -54,12 +54,11 @@ public function __construct($scale = 2, $grouping = true, $roundingMode = self::
      */
     public function transform($value)
     {
-        if (null !== $value) {
+        if (null !== $value && 1 !== $this->divisor) {
             if (!is_numeric($value)) {
                 throw new TransformationFailedException('Expected a numeric.');
             }
-
-            $value /= $this->divisor;
+            $value = (string) ($value / $this->divisor);
```

Comment:
This looks wrong to me. Previously, `$value` was a float here, but now it is a string.

Reply:
I changed it from `(float) (string) ($value / $this->divisor);` to `(string) ($value / $this->divisor);` because this functions purpose is to _Transforms a normalized format into a localized money string._ according to the PHPDoc-Block. The return type is also string so I changed it. Should I put a float cast in front of it again?

Reply:
that is fine. the parent transform should be able handle both string or float.